base/commands/map/map_destroy.go

```diff
@@ -0,0 +1,65 @@
+package _map
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-go-client"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/prompt"
+)
+
+const (
+	mapDestroyFlagAutoYes = "yes"
+)
+
+type MapDestroyCommand struct{}
+
+func (mc *MapDestroyCommand) Init(cc plug.InitContext) error {
+	help := "Destroy a Map"
+	cc.SetCommandHelp(help, help)
```

Comment:
Maybe we can add more information in the long help. Something like:
```
Destroy a Map

This command will delete the Map and the data in it will not be available anymore.
```